.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -103,9 +104,9 @@ body:
       description: What package manager are you using?
       multiple: true
       options:
-        - npm 8
-        - npm 7
+        - npm 10
         - npm 6
```

Comment:
keeping npm 6 right now, IIRC that was the last version with the old peerDependency rules.
Open to removing it and focusing only on newer npm